### DISCUSSION TITLE
Is IP Adapter Tiled functionally the same as the other IP Adapter nodes, but for non-square?

### DISCUSSION THREAD

**Discussion by ewebgh33** [cubiq/ComfyUI_IPAdapter_plus#716] [Q&A] [answered]
I ask because I thought I should be using either IP Adapter Advanced or IP Adapter Precise Style/Composition...
But then I need tiled due to non-square aspect, and if I select the option for precise style, is this functionally the same as using an "Ip Adapter Precise Style Transfer" node?

Are the precise style/comp nodes just normal/advanced nodes with the right options already preset?

Basically nothing I see posted on Reddit/CivitAI is square, so what is the standard workflow for using Advanced settings and precise transfer for non-square aspect ratios please?
Just a bit confused if all the variations of the nodes are really that different or the same effect can be achieved all from a tiled node.

**Comment by ewebgh33** [comment]
Also say for example trying out the CosXL workflow, does everything need to be square? Because it's not like I'm pumping in a square of empty latent of set dimensions in that flow (from the examples folder)?

**Comment by cubiq** [comment, marked as answer]
precise style transfer doesn't need a square image.

you use the tiled ipadapter only for linear transfer mostly
